ui/components/multichain/pages/connections/connections.tsx

```diff
@@ -206,50 +192,10 @@ export const Connections = () => {
       data-testid="connections-page"
       className="main-container connections-page"
     >
-      <Header
-        backgroundColor={BackgroundColor.backgroundDefault}
-        startAccessory={
-          <ButtonIcon
-            ariaLabel={t('back')}
-            iconName={IconName.ArrowLeft}
-            className="connections-header__start-accessory"
-            color={IconColor.iconDefault}
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            onClick={() => (history as any).goBack()}
-            size={ButtonIconSize.Sm}
-          />
-        }
-      >
-        <Box
-          display={Display.Flex}
-          alignItems={AlignItems.center}
-          gap={2}
-          justifyContent={JustifyContent.center}
-          className="connections-header__title"
-        >
-          {connectedSubjectsMetadata?.iconUrl ? (
-            <AvatarFavicon
-              name={connectedSubjectsMetadata.name}
-              size={AvatarFaviconSize.Sm}
-              src={connectedSubjectsMetadata.iconUrl}
-            />
-          ) : (
-            <Icon
-              name={IconName.Global}
-              size={IconSize.Sm}
-              color={IconColor.iconDefault}
-            />
-          )}
-          <Text
-            as="span"
-            variant={TextVariant.headingMd}
-            textAlign={TextAlign.Center}
-            ellipsis
-          >
-            {getURLHost(securedOrigin)}
-          </Text>
-        </Box>
-      </Header>
+      <PermissionsHeader
+        securedOrigin={securedOrigin}
+        connectedSubjectsMetadata={connectedSubjectsMetadata}
```

Comment:
Is this a related change?

Reply:
Nice catch, no! this file shouldn't be changed. Updated https://github.com/MetaMask/metamask-extension/pull/26635/commits/76808f645e610095ca5c83e83c9d76445db545db